css-three-column-layout/styles.css

```diff
@@ -0,0 +1,52 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #1996da;
+  width: 80%;
+  margin: auto;
+  height: 100vh;
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  padding-top: 10%;
+}
+
+img {
+  width: 50%;
+  padding-top: 25%;
+}
+
+section {
+  flex-basis: calc(100% / 3);
+  align-items: center;
+  margin: 20px;
+}
+
+.slogan {
+  text-align: center;
+  color: white;
+}
+
+.section-title {
+  color: white;
+}
+
+.container {
+  display: flex;
+  text-align: center;
+}
```

Comment:
`display: flex` in a layout class system should be the `row` clas. I see you are missing one, so you should make this the `row` class instead. You should also make a separate class for centering the text, you can add both classes to the element.